back/spec/acceptance/initiatives_spec.rb

```diff
@@ -689,6 +689,21 @@
     end
   end
 
+  patch 'web_api/v1/initiatives/:id/accept_invite' do
+    before do
+      @initiative = create(:initiative)
+      create(:cosponsors_initiative, initiative: @initiative, user: @user)
+    end
+
+    describe 'for initiative with associated cosponsor' do
+      let(:id) { @initiative.id }
+
+      example_request 'cosponsor accepts invitation' do
+        assert_status 204
+      end
```

Comment:
```suggestion
      @cosponsors_initiative = create(:cosponsors_initiative, initiative: @initiative, user: @user)
    end

    describe 'for initiative with associated cosponsor' do
      let(:id) { @initiative.id }

      example 'cosponsor accepts invitation' do
        expect { do_request }
          .to change { @cosponsors_initiative.reload.status }.from('pending').to('accepted')
        assert_status 204
      end
```
But you can ignore it if you don't like it.

Reply:
Good catch! I deleted that assertion by accident.